src/components/icons/platform/Medium.jsx

```diff
@@ -7,6 +7,7 @@ export const MediumIcon = toSquidInkIcon(
         {...defaultStrokeFill}
+        fill="#0A1C3B"
```

Comment:
we try not to use fill directly in `.jsx` files. 
You should change it when using in a component through CSS if it's needed or change `defaultStrokeFill` or create a new object with new settings.